src/x509.c

```diff
@@ -7565,8 +7565,29 @@ int wolfSSL_X509_REVOKED_get_serial_number(RevokedCert* rev,
     return WOLFSSL_SUCCESS;
 }
 
+const WOLFSSL_ASN1_INTEGER* wolfSSL_X509_REVOKED_get0_serial_number(const
+                                                      WOLFSSL_X509_REVOKED *rev)
+{
+    WOLFSSL_ENTER("wolfSSL_X509_REVOKED_get0_serial_number");
+
+    if (rev != NULL) {
+        return rev->serialNumber;
+    }
+    else
+        return NULL;
+}
+
+const WOLFSSL_ASN1_TIME* wolfSSL_X509_REVOKED_get0_revocation_date(const
```

Comment:
For stubs, we have a macro `NO_WOLFSSL_STUB`. This should only be compiled in `#ifndef NO_WOLFSSL_STUB`.

Reply:
Yes. That is already there in the PR. Or should be:

```
#ifndef NO_WOLFSSL_STUB
const WOLFSSL_ASN1_TIME* wolfSSL_X509_REVOKED_get0_revocation_date(const
                                                      WOLFSSL_X509_REVOKED *rev)
{
    WOLFSSL_STUB("wolfSSL_X509_REVOKED_get0_revocation_date");

    (void) rev;
    return NULL;
}
#endif
```

Reply:
I'm not seeing that in the diff here, nor when I pull down the patch of this PR directly.